account-kit/smart-contracts/src/ma-v2/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { type Hex, concat } from "viem";
+
+export type PackSignatureParams = {
+  // orderedHookData: HookData[];
+  validationSignature: Hex;
+};
+
+// Signature packing utility
+export const packSignature = ({
+  // orderedHookData, TO DO: integrate in next iteration of MAv2 sdk
+  validationSignature,
+}: PackSignatureParams): Hex => {
+  return concat(["0xFF", "0x00", validationSignature]);
+};
+
+export const addresses = {
+  allowlistModule:
+    "0xE46ca4a98c485caEE2Abb6ef5116292B8c78a868" as `0x${string}`,
+  nativeTokenLimitModule:
+    "0xEa6a05306315196f2A7CA2ec7eEA45290bae00A0" as `0x${string}`,
+  paymasterGuardModule:
+    "0x976D01aF75D128cae526B5328AC268ac83D607f4" as `0x${string}`,
+  singleSignerValidationModule:
+    "0xF56716aE104545BdAf012e14e4640beb52727479" as `0x${string}`,
+  timeRangeModule:
+    "0x6FD0a9765a86788126a55aD5a483029a484F996C" as `0x${string}`,
+  webauthnValidationModule:
+    "0xCf3423F8EB9EE215560802D03a48cDD44f85bD28" as `0x${string}`,
+  executionInstallDelegate:
+    "0x8Bf909fEb66EBcC4725f04E70F319791Ec9d9628" as `0x${string}`,
+  modularAccountImpl:
+    "0x99090abd2700E24Cc70E3c486A89F7af876fFA33" as `0x${string}`,
+  semiModularAccountBytecodeImpl:
+    "0xDCBb5d4639428B18703801f7Cd7230add729E4b0" as `0x${string}`,
+  semiModularAccountStorageOnlyImpl:
+    "0x563ea43Ba0CD9150466B054Ce47FB0Fb45B234E2" as `0x${string}`,
+  accountFactory: "0xA5FC7aDc6d2c838443dF5bb826152c58b918f2c8" as `0x${string}`,
+  accountFactoryOwner:
+    "0x3BD786ac7Dec96eF9d06ebbCFa4e0E0947A2c303" as `0x${string}`,
+};
```

Comment:
would love to have these live more closely to the definition. this big list will become a pain to maintain